qiskit_experiments/framework/composite/composite_experiment.py

```diff
@@ -131,3 +132,75 @@ def _postprocess_transpiled_circuits(self, circuits, **run_options):
         for expr in self._experiments:
             if not isinstance(expr, CompositeExperiment):
                 expr._postprocess_transpiled_circuits(circuits, **run_options)
+
+
+def sync_transpile_options(
```

Comment:
Why is this needed in the first place? I wonder if this should be the default behavior of `CompositeExperiment`? Or why not have this as an extra function in the `CompositeExperiment`? I don't see why the decorator framework is needed when we only decorate a single method of a single class.

Reply:
This aims at making a batch experiment to behave as if a single experiment (because user doesn't need to recognize its implementation to use). This SHOUD not be default behavior (unfortunately it is now) because we should be able to transpile batch (I believe also parallel) experiment with its own configuration.